docs/conf.py

```diff
@@ -51,3 +50,4 @@
 source_suffix = '.rst'
 html_css_files = ["custom.css"]
 html_favicon = "sources/icon.svg"
+pygments_style = 'sphinx'
```

Comment:
Do we need this?
I noticed that it makes the code background greenish.

Reply:
Removed it